src/content/docs/en/guides/imports.mdx

```diff
@@ -183,16 +193,17 @@ import logoUrl from '@assets/logo.png?url';
 These aliases are also integrated automatically into [VS Code](https://code.visualstudio.com/docs/languages/jsconfig) and other editors.
 
 
-## `Astro.glob()`
+## `import.meta.glob()`
 
-[`Astro.glob()`](/en/reference/api-reference/#astroglob) is a way to import many files at once.
+[Vite's `import.meta.glob()`](https://vitejs.dev/guide/features.html#glob-import) is a way to import many files at once using wildcards to create a list of pathnames.
```

Comment:
I feel like this is technically wrong? Wildcards are something that glob patterns contain to represent different characters, and it doesn't "create" the pathnames, rather it crawls any matching paths on the user's file system and returns them.

```suggestion
[Vite's `import.meta.glob()`](https://vitejs.dev/guide/features.html#glob-import) is a way to import many files at once using glob patterns to find matching file paths.
```